src/views/GovernanceOverview/index.tsx

```diff
@@ -0,0 +1,165 @@
+import React from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import Header from 'components/Header';
+import './index.scss';
+import CollapsableLink from 'components/shared/CollapsableLink';
+import Button from 'components/shared/Button';
+
+type NumberedListItemProps = {
+  index: string;
+  header: string;
+  body: string;
+};
+
+const NumberedListContinuingItem = ({
+  index,
+  header,
+  body
+}: NumberedListItemProps) => {
+  return (
+    <div className="easi-governance-overview__numbered-list-item">
+      <div className="easi-governance-overview__list-item-continuing">
+        <span className="easi-governance-overview__dot">{index}</span>
+      </div>
+      <div className="margin-top-05 margin-left-1 margin-bottom-3 tablet:margin-bottom-4">
+        <p className="text-bold margin-top-0 margin-bottom-05">{header}</p>
+        <p className="margin-bottom-0 line-height-body-4 margin-top-2px">
+          {body}
+        </p>
+      </div>
+    </div>
+  );
+};
+
+const NumberedListTerminalItem = ({
+  index,
+  header,
+  body
+}: NumberedListItemProps) => {
+  return (
+    <div className="easi-governance-overview__numbered-list-item">
+      <div className="easi-governance-overview__list-item-terminal">
+        <span className="easi-governance-overview__dot">{index}</span>
+      </div>
+      <div className="margin-top-05 margin-left-1 margin-bottom-0">
+        <p className="text-bold margin-top-0 margin-bottom-05">{header}</p>
+        <p className="margin-bottom-0 line-height-body-4 margin-top-2px">
+          {body}
+        </p>
+      </div>
+    </div>
+  );
+};
+
+const GovernanceOverview = ({ history }: RouteComponentProps) => {
+  return (
+    <div className="easi-governance-overview">
+      <Header name="EASi Governance Overview" />
+      <main role="main" className="grid-container">
+        <p>
+          <a href="/">Home</a> <i className="fa fa-angle-right" /> Add a new
```

Comment:
🔧  It would be better to use `Link` from ReactRouter here, instead of a naked `a` tag, so we don't force the browser to reload the app context. That will help avoid, for example, the login page flashing, etc.